build/pause/Makefile

```diff
@@ -56,6 +58,9 @@ endif
 # If you want to build AND push all containers, see the 'all-push' rule.
 all: all-container
 
+push-manifest: all-container all-push manifest-tool
```

Comment:
I'd call this `release`, or rename `all-push` into something else and call this one `all-push`